core/domain/collection_services.py

```diff
@@ -1122,7 +1122,13 @@ def get_next_page_of_all_commits(
 
     Returns:
         3-tuple of (results, cursor, more) as described in fetch_page() at:
-        https://developers.google.com/appengine/docs/python/ndb/queryclass
+        https://developers.google.com/appengine/docs/python/ndb/queryclass,
+        where:
+            results: list(CollectionCommitLogEntry). List of query results.
```

Comment:
The code below seems to suggest CollectionCommitLogEntry `([collection_domain.CollectionCommitLogEntry(
        entry.created_on, entry.last_updated, entry.user_id, entry.username,
        entry.collection_id, entry.commit_type, entry.commit_message,
        entry.commit_cmds, entry.version, entry.post_commit_status,
        entry.post_commit_community_owned, entry.post_commit_is_private
    ) for entry in results], new_urlsafe_start_cursor, more)`